tests/test_imports.py

```diff
@@ -0,0 +1,54 @@
+import os
+import imp
+import pkgutil
+import sys
+import unittest
+
+
+class ImportTestSuite(unittest.TestCase):
+
+    def test_import_of_each_module(self):
+        """Import each module of package git_cc.
+
+        This test was added because the import Python 3 requires you to use the
+        relative package import syntax "from . import <module name>" to import
+        packages from the same package, whereas Python 2 you could just use
+        "import <module name>".
```

Comment:
It is not too bad. It only checks whether git-cc modules, existing or new, import other git-cc modules in a way that does not break Python 3 compatibility. No module names are hard-coded, which turned out to be the difficult part of the test.